docs/alerts/understanding-alerts.md

```diff
@@ -0,0 +1,32 @@
+# 1. Understanding Alerts in Netdata
+
+In Netdata, an **alert** (also called a "health check") is a rule that continuously monitors one or more metrics and assigns a **status** based on whether conditions are met.
+
+| Core Concept | Description |
+|--------------|-------------|
+| **Where alerts are evaluated** | Locally on each Netdata Agent or Parent node, not centrally in Netdata Cloud |
```

Comment:
I'd phrase this as "Locally on an Agent, [..]". The description below explains in more detail.